src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_vm_diagnostics_templates.py

```diff
@@ -278,248 +278,702 @@ def get_default_diag_config(is_windows):
     else:
         return {
             "ladCfg": {
```

Comment:
Thanks for your info. Followed your lead and regenerated the new recording and pushed the 3rd commit with that change.